x-pack/plugins/ingest_manager/server/services/epm/packages/get.ts

```diff
@@ -86,7 +86,7 @@ export async function getPackageKeysByStatus(
   savedObjectsClient: SavedObjectsClientContract,
   status: InstallationStatus
 ) {
-  const allPackages = await getPackages({ savedObjectsClient });
+  const allPackages = await getPackages({ savedObjectsClient, experimental: true });
```

Comment:
Are we adding `experimental` to match another query somewhere else, or is it a coincidence?

If it should be in sync with somewhere else, it'd be good to document that in a comment or share the function/options.

LGTM either way. Thanks for the tests!

Reply:
@jfsiii It's being added as a search param to the `registry.fetchList` called by `getPackages`.  Essentially: `https://epr.elastic.co/search?experimental=true`.  `getPackages` is typed to accept `searchParams`.  Is that sufficient or were you meaning something else?

Reply:
Something else, but after re-reading the issue & this code there's no need to make any changes. 

I was going off my recollection of #80032. I thought there might have been a situation where this flag needed to be the same value as another call. i.e. "we called A with true, but B with false. both calls should use the same value, both true or both false".

I see now see "A" is `Registry.fetchList` and "B" is this updated `getPackages` which calls "A" with that param so they're already linked. Sorry for the noise.